### ISSUE TITLE
Change lookup init to use static arrays

### ISSUE THREAD

**Issue by chrispcampbell** [climateinteractive/SDEverywhere#18]
When investigating performance/crashing issues with En-ROADS on iPad, I discovered that a large part of the memory pressure is coming from the way the lookup initialization code gets compiled down to wasm.

The current approach passes a list of (x,y) pairs as varargs to `__new_lookup`, which then copies them into a freshly malloc'd array in the `Lookup`.  Something about the way Emscripten handles the varargs seems to cause higher memory usage when the module is loaded and also increases the on-disk size of the wasm file.

An example init line currently looks like this:
```
__lookup1 = __new_lookup(111, 1990.0, 7.53332, 1991.0, 7.7605, 1993.0, 8.15487, ..., 2100.0, 0.550535);
```

Some rough baseline performance stats from my ancient (2013) iPad Air:
```
wasm on disk size: 1084036 bytes
init time: 28267 ms
memory use after init: 45 MB (JS) / 253 MB (page)
memory use after running model: 45 MB (JS) / 450 MB (page)
```

(The memory usage stats here and below are fuzzy, representing the rough max values reported using Safari developer tools.)

I tried a couple alternatives:

1. Make `__new_lookup` take an explicit `double *`, then pass a literal array, for example:
```
__lookup1 = __new_lookup(111, (double[]){ 1990.0, 7.53332, 1991.0, 7.7605, 1993.0, 8.15487, ..., 2100.0, 0.550535 });
```

This made a big improvement:
```
wasm on disk size: 775426 bytes
init time: 4469 ms
memory use after init: 81 MB (JS) / 124 MB (page)
```

2. Make `__new_lookup` take an explicit `double *`, then pass a static array declared earlier, for example:
```
const double __lookup1_data[222] = { 1990.0, 7.53332, 1991.0, 7.7605, 1993.0, 8.15487, ..., 2100.0, 0.550535 };
...
__lookup1 = __new_lookup(111, /*copy=*/false, __lookup1_data);
```

This made another big dent:
```
wasm on disk size: 773968 bytes
init time: 4046 ms
memory use after init: 64 MB (JS) / 103 MB (page)
```

**Comment by chrispcampbell** [comment]
Here are some performance stats for this change.  Additional improvements will come in #19.

### Performance

__MacBook Pro (2019)__ | _2.4 GHz 8-core i9, 32 GB RAM, macOS 10.15_

| Issue | C run (ms) | Wasm run (ms) | Wasm init (ms) | JS mem (MB) | Page mem (MB) |
| ----- | ---------: | ------------: | -------------: | ----------: | ------------: |
| baseline | 45.8 | 87.5 | 38.0 | 94 | 685 |
| SDE 18 | 46.0 | 85.6 | 18.0 | 39 | 672 |

__iPhone 8__ | _A11, iOS 13_

| Issue | C run (ms) | Wasm run (ms) | Wasm init (ms) | JS mem (MB) | Page mem (MB) |
| ----- | ---------: | ------------: | -------------: | ----------: | ------------: |
| baseline | 39.9 | 187.0 | 165.0 | 39 | 645 |
| SDE 18 | 40.3 | 219.0 | 86.0 | 38 | 724 |

__iPad Air (2013)__ | _A7, iOS 12_

| Issue | C run (ms) | Wasm run (ms) | Wasm init (ms) | JS mem (MB) | Page mem (MB) |
| ----- | ---------: | ------------: | -------------: | ----------: | ------------: |
| baseline | 151.0 | 1372.2 | 30146.0 | 77 | 331 |
| SDE 18 | 166.0 | 1408.0 | 4416.0 | 42 | 395 |

### Size

| Issue | Wasm size (bytes) |
| ----- | ----------------: |
| baseline | 1,084,036 |
| SDE 18 | 773,968 |

**Comment by chrispcampbell** [comment]
Merged to `develop` in 3c6c0a4.
